draft-ietf-rats-eat.md

```diff
@@ -510,29 +510,30 @@ independent of encoding.  Each claim is defined as a CDDL group.
 In {{encoding}} on encoding, the CDDL groups turn into CBOR map entries and JSON name/value pairs.
 
 Each claim described has a unique text string and integer that identifies it.
-CBOR-encoded tokens MUST use only the integer for Claim Keys.
-JSON-encoded tokens MUST use only the text string for Claim Names.
+CBOR-encoded tokens MUST use only the integer for claim keys.
+JSON-encoded tokens MUST use only the text string for claim names.
 
 
 
-## EAT Nonce Claim (nonce) {#nonce}
+## eat_nonce (EAT Nonce) Claim {#nonce}
 
 An EAT nonce is either a byte or text string or an array of byte or text strings.
 The array option supports multistage EAT verification and consumption.
 
 A claim named "nonce" was defined and registered with IANA for JWT, but MUST NOT be used because it does not support multiple nonces.
-No previous nonce claim was defined for CWT.
-To distinguish from the previously defined nonce claim, this claim is named "eat_nonce" in JSON-encoded EATs.
+No previous "nonce" claim was defined for CWT.
+To distinguish from the previously defined JWT "nonce" claim, this claim is named "eat_nonce" in JSON-encoded EATs. The CWT nonce defined
+here is intended for general purpose use and retains the "Nonce" claim name instead of an EAT-specific name.
```

Comment:
This one and the one below were on purpose since the CWT claim in not eat_nonce. Shall I change or leave (see the email exchange with Giri re: renaming this and Profile from earlier this week)?